src/mumble/GlobalShortcut_unix.cpp

```diff
@@ -54,9 +56,12 @@ GlobalShortcutEngine *GlobalShortcutEngine::platformInit() {
 }
 
 GlobalShortcutX::GlobalShortcutX() {
+#ifdef NO_X11
 	iXIopcode = -1;
 	bRunning  = false;
 
+	return;
```

Comment:
I think this `return` might come as a surprise in future revisions as the `#else` block doesn't end with a return statement. Therefore one might add something to the function's end that should be executed in all cases but if `NO_X11` is set, then it won't be executed due to this `return`.

Imo we're better off without this here :thinking: